pom.xml

```diff
@@ -321,6 +320,21 @@
                 <artifactId>jib-maven-plugin</artifactId>
                 <version>${jib-maven-plugin.version}</version>
             </plugin>
+
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>${jib-maven-plugin.version}</version>
+                <configuration>
+                    <containerizingMode>packaged</containerizingMode>
```

Comment:
Is a packaged mode necessary?

Reply:
it's not as far as I can tell, have tested it locally without seems to work fine, is without packaged mode in my PR [here](https://github.com/companieshouse/psc-data-api/pull/120)

Reply:
Changes merged from main has fixed this.